smartmin/views.py

```diff
@@ -141,10 +141,6 @@ def lookup_obj_attribute(self, obj, field):
         # next up is the object itself
         obj_field = getattr(obj, curr_field, None)
 
-        # if it is callable, do so
-        if obj_field and getattr(obj_field, '__call__', None):
```

Comment:
You got an example of us using that - I honestly don't recall seeing that. What we do a lot is add a method to the view like `get_foo`.

Problem is things are callable that we don't actually want to call. Maybe we limit to bound methods with no params?

https://stackoverflow.com/questions/53225/how-do-you-check-whether-a-python-method-is-bound-or-not